core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.css

```diff
@@ -87,7 +87,7 @@
 }
 .oppia-sidebar-menu-open {
   box-shadow: 1px 0 3px rgba(0,0,0,0.12), 1px 0 2px rgba(0,0,0,0.24);
-  height: 100vh;
+  height: 100vh; 
```

Comment:
Please drop the extra space that you added at the end of this line. (In general, look at the "Files Changed" tab on GitHub to verify your fixes before you send them for review.)

Reply:
Done !!!